MIGRATION.md

```diff
@@ -1,6 +1,201 @@
 # Introduction
 
 When new releases include breaking changes or deprecations, this document describes how to migrate.
+## Migrating to 1.4.0
+
+### Deprecations
+
+- The `dagit` python package and all references to it are now deprecated. We will continue to publish `dagit` and support APIs that used the term “dagit” until v2.0, but you should transition to newer `dagster-webserver` package. This is a drop-in replacement for `dagit`. Like `dagit`, it exposes an executable of the same name as the package itself, i.e. `dagster-webserver`.
+    - Any Dockerfiles or other Python environment specifications now list `dagster-webserver` instead, e.g.:
+
+    ```python
+    ### Dockerfile
+    # ...
+
+    # no (deprecated)
+    RUN pip install dagster dagit ...
+    ...
+    ENTRYPOINT ["dagit", "-h", "0.0.0.0", "-p", "3000"]
+
+    # yes
+    RUN pip install dagster dagster-webserver
+    ...
+    ENTRYPOINT ["dagster-webserver", "-h", "0.0.0.0", "-p", "3000"]
+    ```
+
+    - Three fields containing the term “dagit” in the Dagster helm chart schema have been updated to use “dagsterWebserver” instead. Old `values.yaml`:
+
+        ```python
+        ### values.yaml
+
+        # no (deprecated)
+        dagit:
+          ...
+          # ...
+        ingress:
+          dagit: ...
+          readOnlyDagit: ...
+
+        # yes
+        dagsterWebserver:
+          ...
+          # ...
+        ingress:
+          dagsterWebserver: ...
+          readOnlyDagsterWebserver: ...
+        ```
+
+- We’ve deprecated the `non_argument_deps` parameter of `@asset` and `@multi_asset` in favor of a new `deps` parameter. To update your code to use `deps`, simply rename any instances of `non_argument_deps` to `deps` and change the type from a set to list. Additionally, you may also want to begin passing the python symbols for assets, rather than their `AssetKey`s to improve in-editor experience with type-aheads and linting.
+
+```python
```

Comment:
It looks like code blocks higher up in the doc are indented, but this and some of the lower ones aren't.  I don't have a super strong opinion on which is correct, but we should be consistent.